modules/ROOT/Edits/endpoint-sftp-receive-target.adoc

```diff
@@ -0,0 +1,77 @@
+= SFTP Receive Endpoint Settings
+
+SFTP receive endpoint settings determine where to receive messages in the following contexts:
+
+* Receive from partners
++
+Source endpoint in an inbound message flow that receives B2B/EDI messages from partners either directly or via 3rd party connections
++
+* Source at host
++
+Source endpoint in an outbound message flow that receives application JSON and XML messages from the backend systems
+
+A host, partner, or 3rd party connection can own FTP send endpoints.
+
+Create SFTP receive endpoints by configuring these settings on the host or Partner page:
+
+[%header%autowidth.spread]
+|===
+|Name |Description | Required
+| Endpoint Name
+| The automatically generated name used to identify the endpoint, which you can subsequently modify manually.
```

Comment:
Slightly different wording that used previously in same context; I like this one better.